dask/dataframe/core.py

```diff
@@ -5574,6 +5596,7 @@ def map_partitions(
     $META
     """
     name = kwargs.pop("token", None)
+    parent_meta = kwargs.pop("parent_meta", None)
```

Comment:
This would be similar to the above one, when `_Frame` list can be all empty or the args has no `_Frame` at all and the `meta` at hand is way too generic.